roles/gitea/defaults/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+#gitea_install: True
+#gitea_enabled: True
+
```

Comment:
Plz structure defaults/main.yml in the same way as:
https://github.com/iiab/iiab/blob/master/roles/lokole/defaults/main.yml

Specifically, add these 2 lines as separator, to delineate the stuff (above it!) that's already defined in vars/default_vars.yml:

# All above are set in: github.com/iiab/iiab/blob/master/vars/default_vars.yml
# If nec, change them by editing /etc/iiab/local_vars.yml prior to installing!